src/components/widget/templates/mangrove-coverage/component.js

```diff
@@ -60,65 +65,68 @@ class MangroveCoverage extends React.PureComponent {
   }
 
   render() {
-    const { data, currentLocation, slug } = this.props;
-    const { chartConfig, metadata } = data;
+    const { data: { chartConfig, metadata }, currentLocation, slug } = this.props;
     const { currentYear, unit } = this.state;
     const optionsYears = metadata.years.map(year => ({
       label: year.toString(),
       value: year
     }));
-    const widgetData = this.getData();
-    const { percentage } = widgetData[0];
-    const unitOptions = [
-      { value: '%', label: '%' },
-      { value: 'km', label: 'Km' }
-    ];
-    const totalCoverage = metadata.total / 1000;
-    const coverage = (percentage * totalCoverage) / 100;
+    let content = null;
 
-    return (
-      <Fragment>
-        <div className={styles.widget_template}>
+    try {
+      const widgetData = this.getData();
+      const { percentage } = widgetData[0];
+      const unitOptions = [
+        { value: '%', label: '%' },
+        { value: 'km', label: 'Km' }
```

Comment:
We need to change that label into km2 as well, we can use <DangerousHTML html={label} /> , we can do that in another task if you prefer

Reply:
I think for this widget is linear quantity so km is fine... it is coastline, not mangrove area. We can confirm that with Ed or Lammer